packages/flutter_tools/test/general.shard/android/android_install_test.dart

```diff
@@ -82,7 +82,7 @@ void main() {
       processManager: processManager,
     );
 
-    expect(await androidDevice.installApp(androidApk), false);
+    expect(await androidDevice.installApp(androidApk), isFalse);
```

Comment:
In my opinion, this change is not worth generating a diff. `isTrue`/`isFalse` don't generate better failure output, and I don't find this line of code to be significantly more readable than the original[^1].

See https://github.com/dart-lang/test/issues/2358

[^1]: If it is decided to be more readable, then we would have to consider using the `equals` matcher in place of bare values throughout the rest of the tool code.

Reply:
TIL